resources/views/posts/show.blade.php

```diff
@@ -32,7 +32,7 @@ function get_posts(){
                 })
                 .catch(function (error){
                     //handle error
-                    console.log(error);
+                    alert(error);
```

Comment:
Kindly check the error data return by the api. From there, you can extract the validation errors. 
https://gyazo.com/c39108bf5f312f4f16e778ccbbae55a4